Lib/ntpath.py

```diff
@@ -640,23 +638,65 @@ def commonpath(paths):
         genericpath._check_arg_types('commonpath', *paths)
         raise
 
+MAX_PATH = 260
```

Comment:
`_INTERNAL_CONSTANT` or else document it (as a legacy value that can largely be ignored)

Reply:
My later suggestion may remove the need for this constant entirely.

Reply:
I think it's useful to document as a public constant, but maybe change it to `MAX_PATH_DOS` or `MAX_PATH_LEGACY`? Python still has an inter-operation concern even if our own process supports long paths (in addition to our manifest setting, this requires enabling at the OS level , which last I checked isn't the default setting). Along the same lines, `CreateProcess` doesn't allow the current directory (explicit or inherited) to be a long path.

Reply:
We've never had it in the past, and it's becoming less relevant. I'd rather not add it now, and continue the work we do to handle it as transparently as possible.